public/plugin.tsx

```diff
@@ -100,6 +102,26 @@ export class CustomImportMapPlugin
       },
     });
 
+    core.chrome.navGroup.addNavLinksToGroup(DEFAULT_NAV_GROUPS.observability, [{
+      id: MAPS_APP_ID,
+      category: DEFAULT_APP_CATEGORIES.dashboardAndReport,
+      order: 200,
+    }]);
+
+    core.chrome.navGroup.addNavLinksToGroup(DEFAULT_NAV_GROUPS.analytics, [{
+      id: MAPS_APP_ID,
+      category: DEFAULT_APP_CATEGORIES.dashboardAndReport,
+      order: 200,
+      // Pending change ready from OSD
+      // showInAllNavGroup: true,
+    }]);
```

Comment:
Hi Junqiu, confirmed that analytics won't have Maps, could you please add Maps to security analytics?

Reply:
Ack, thanks confirm!

Reply:
Then do we still need put 
```
// Pending change ready from OSD
// showInAllNavGroup: true,
```
for the `security analytics` section? @SuZhou-Joe

Reply:
Ack, we can add it latter once confirmed